### ISSUE TITLE
Improve login flow

### ISSUE THREAD

**Issue by MoazAlkharfan** [ChilliCream/hotchocolate#5526]
### Is your feature request related to a problem?

Not exactly a problem.

The current login window which opens is a separate window from the imbedded browser.
What's been bothering me about it is that I'm already logged-in through the default browser, and I need to enter the credentials often as the login expires often, at the same time while using a password manager which is also a pain to get into and get the password to use in the imbedded browser login form.

In short, I need to get the password then write the credentials then approve the login because of 2FA quite often.

### The solution you'd like

The login window should be using the default system browser which in term shortens the login flow to a few clicks as I'm already logged in to these services in the default system browser.

### Product

Banana Cake Pop

**Comment by rstaib** [comment]
@MoazAlkharfan thanks for the feedback. I guess you're talking about the app, right?

**Comment by MoazAlkharfan** [comment]
@rstaib Sorry for the late reply. Yes exactly, the app.

**Comment by nesherhh** [comment]
Having same problem. 1Password or any password manager extension does not work in login window
